CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Other Changes
 
 * Added the `Waypoint.layer` property, which can ensure that the route begins on the correct road if it is above or below another road. ([#745](https://github.com/mapbox/mapbox-directions-swift/pull/745))
+* Expanded `AttributeOptions` to allow user options with custom values. See `CustomValueOptionSet` for reference. ([#748](https://github.com/mapbox/mapbox-directions-swift/pull/748))
```

Comment:
`CustomValueOptionSet` is probably more interesting to us internally; from a developer standpoint, the one to pay attention to is `AttributeOptions.customOptionsByRawValue`.